image/mirror.go

```diff
@@ -1,20 +1,23 @@
 package image
 
-import "strings"
+import (
+	"fmt"
+	"strings"
+)
 
 var Mirrors = map[string]string{}
 
 func Mirror(image string) string {
 	orig := image
-	if strings.HasPrefix(image, "weaveworks") {
+	if strings.HasPrefix(image, "weaveworks") || strings.HasPrefix(image, "registry") || strings.HasPrefix(image, "busybox") {
```

Comment:
Is there a ticket for this fix? Want to understand the usecase for returning image name as-is in these additional cases

Reply:
We have an exclude in the code for `weaveworks` but not for the others that we don't do any string manipulation, this change makes it so people reading the code understand what we don't manipulate.